main.go

```diff
@@ -53,6 +55,8 @@ func main() {
 	fs.StringVar(&dbUser, "db-user", "postgres", "database username")
 	fs.StringVar(&dbPass, "db-pass", "", "database user password")
 	fs.StringVar(&dbURL, "database-url", "", "database connection URL")
+	fs.StringVar(&metricsTopic, "metrics-topic", "platform.upload.client-metrics", "topic on which to place metrics data")
```

Comment:
I prefer to default values to something generic, or an empty string. We can always force deployed software to take values from flags or environment variables. But defaulting to values more appropriate to development execution means less clutter in your local development environment.